app/assets/javascripts/angular/filters/hyphens.coffee

```diff
@@ -0,0 +1,4 @@
+angular.module "lean-coffee"
+  .filter 'hyphens', ($sce) ->
+    (str) ->
+      str.replace(/\s+/g, '-')
```

Comment:
Cool. Should i create a new commit with this on or is it just for the next time?